src/HttpKernel/ContaoKernel.php

```diff
@@ -108,15 +108,7 @@ public function getLogDir(): string
     public function getPluginLoader(): PluginLoader
     {
         if (null === $this->pluginLoader) {
-            $this->pluginLoader = new PluginLoader($this->getProjectDir().'/vendor/composer/installed.json');
-
-            $config = $this->getManagerConfig()->all();
-
-            if (isset($config['contao_manager']['disabled_packages'])
-                && \is_array($config['contao_manager']['disabled_packages'])
-            ) {
-                $this->pluginLoader->setDisabledPackages($config['contao_manager']['disabled_packages']);
-            }
+            $this->pluginLoader = new PluginLoader();
```

Comment:
Please revert these changes. They are not related to the dev access key changes and I cannot merge this PR with them in.